examples/weather/unified_recipe/train.py

```diff
@@ -103,7 +105,6 @@ def main(cfg: DictConfig) -> None:
             },  # TODO: maybe mobe this conversion to resolver?
```

Comment:
Note: `rank_zero_logger` is not used. Probably should make it consistent with usage of `LaunchLogger`?